e2e/playwright/tests/test.list.ts

```diff
@@ -6,4 +6,8 @@ import core from './todo_plugin.spec';
 
 import '../support/init_test';
 
-test.describe(core.connected);
+// Test if plugin is setup correctly
+test.describe("setup", core.setup);
+
+// Test various plugin actions
+test.describe("actions", core.commands);
```

Comment:
I think this may read a bit better. It definitely would be nice to be able to pick and choose which commands will be tested from this file

```suggestion
// Test various plugin actions
test.describe('commands', commands.help);
```